sdk/easm/Azure.Analytics.Defender.Easm/README.md

```diff
@@ -0,0 +1,177 @@
+[Diagnostics](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/Diagnostics.md) |
+[Mocking](https://learn.microsoft.com/dotnet/azure/sdk/unit-testing-mocking) |
+[Client lifetime](https://devblogs.microsoft.com/azure-sdk/lifetime-management-and-thread-safety-guarantees-of-azure-sdk-net-clients/)
+<!-- CLIENT COMMON BAR -->
+
+## Examples
+
+You can familiarize yourself with different APIs using [Samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/easm/Azure.Analytics.Defender.Easm/samples).
+
+### View Assets
+
+You can call the client's `AssetsList` method to view your assets.
+
+```C# Snippet:Sample1_AssetResources_Get_Assets
+var response = client.GetAssetResources();
+int index = 0;
+foreach (AssetResource asset in response)
+{
+    Console.WriteLine($"Asset Name: {asset.Name}, Kind: {asset.GetType()}");
+    if (index++ > 5)
+    {
+        break;
+    }
+}
+```
+
+### Create Discovery Group
+ 
+You can call the client's `DiscoGroupCreateOrReplace` method to create or replace your disco group
+
+```C# Snippet:Sample2_DiscoveryGroups_Create_Discovery_Group
+string discoveryGroupName = "Sample Disco From C#";
+string discoveryGroupDescription = "This is a sample discovery group generated from C#";
+            string[] hosts = new string[2];
+hosts[0] = "<host1>.com";
+hosts[1] = "<host2>.com";
+string[] domains = new string[2];
+domains[0] = "<domain1>.com";
+domains[1] = "<domain2>.com";
+                        DiscoveryGroupPayload request = new DiscoveryGroupPayload();
+foreach (var host in hosts)
+{
+    DiscoverySource seed = new DiscoverySource();
+    seed.Kind = DiscoverySourceKind.Host;
+    seed.Name = host;
+    request.Seeds.Add(seed);
+}
+foreach (var domain in domains)
+{
+    DiscoverySource seed = new DiscoverySource();
+    seed.Kind = DiscoverySourceKind.Domain;
+    seed.Name = domain;
+    request.Seeds.Add(seed);
+}
+
+request.Description = discoveryGroupDescription;
+client.CreateOrReplaceDiscoveryGroup(discoveryGroupName, request);
+```
+
+### Run the Discovery Group
+
+You can call client's `DiscoGroupRun` method to run a disco group
+
+```C# Snippet:Sample2_DiscoveryGroups_Run
+client.RunDiscoveryGroup(discoveryGroupName);
+Pageable<DiscoveryGroup> response = client.GetDiscoveryGroups();
+foreach (DiscoveryGroup discoGroup in response)
+{
+    Console.WriteLine(discoGroup.Name);
+    Pageable<DiscoveryRunResult> discoRunPageResponse = client.GetDiscoveryGroupRuns(discoGroup.Name);
+    int index = 0;
+    foreach (DiscoveryRunResult discoRun in discoRunPageResponse)
+    {
+        Console.WriteLine($" - started: {discoRun.StartedDate}, finished: {discoRun.CompletedDate}, assets found: {discoRun.TotalAssetsFoundCount}, status: {discoRun.State}");
+        if (++index == 5){
+            break;
+        }
+    }
+}
+```
+
+
+## Troubleshooting
+
+### Enabling Logging
```

Comment:
```suggestion
### Enabling logging

```

The Microsoft style guide requires sentence-style casing.

Reply:
I already pushed this change earlier, it should show up in a bit